src/AppBundle/Tests/Utils/SluggerTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tests\Utils;
+
+use AppBundle\Utils\Slugger;
+
+/**
+ * Unit test for the application utils.
+ * See http://symfony.com/doc/current/book/testing.html#unit-tests
+ *
+ * Execute the application tests using this command (requires PHPUnit to be installed):
+ *
+ *     $ cd your-symfony-project/
+ *     $ phpunit -c app
+ *
+ */
+class SluggerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
```

Comment:
__[development][code_sniffer]__
* Doc comment for parameter "$string" missing
* Doc comment for parameter "$slug" missing

<!-- Comment by [SideCI](https://sideci.com) -->